modules/mri_violations/php/module.class.inc

```diff
@@ -24,117 +24,140 @@ namespace LORIS\mri_violations;
  */
 class Module extends \Module
 {
+    /**
+     * Gets the base query (FROM and WHERE clause) used to retrieve
+     * all the MRI violations for a given user.
+     *
+     * @return void
+     */
+    public static function getMriViolatedScanQuery()
+    {
+        // Find which sites the user has access to
+        $accessibleSites = \User::singleton()->hasPermission('access_all_profiles')
```

Comment:
I guess the permission needs to be changed to the mri_violation_all_site permission here. No?

Reply:
Correct